lib/matplotlib/text.py

```diff
@@ -1945,10 +1948,7 @@ def _update_position_xytext(self, renderer, xy_pixel):
 
     @artist.allow_rasterization
     def draw(self, renderer):
-        """
-        Draw the :class:`Annotation` object to the given *renderer*.
-        """
-
+        """Draw the `Annotation` object to the given *renderer*."""
```

Comment:
inherit docstring?

Reply:
The inherited one would be "Draw the `Text` object to the given *renderer*.", which is slightly confusing. Right now, I can't think of a generic version that would suit both. So would leave this in.

Reply:
Text.draw could also be inherited, I guess?

Reply:
The base is [Artist.draw](https://matplotlib.org/devdocs/api/_as_gen/matplotlib.artist.Artist.draw.html), which has a technical comment on subclassing. It's probably possible to find a generic way of expressing that, so that it's appropriate in all Artist subclasses. But for now, I think leaving the explicit docstrings is the simplest solution.

Reply:
I guess I'll fix the base docstring later.